packages/x-components/src/x-modules/queries-preview/events.types.ts

```diff
@@ -1,7 +1,20 @@
+import { SearchRequest } from '@empathyco/x-types';
+
 /**
  * Dictionary of the events of QueriesPreview XModule, where each key is the event name, and the
  * value is the event payload type or `void` if it has no payload.
  *
  * @public
  */
-export interface QueriesPreviewXEvents {}
+export interface QueriesPreviewXEvents {
+  /**
+   * Query preview request have been changed.
```

Comment:
```suggestion
   * Any property of the queries preview request has changed.
```

To match other modules